openassessment/xblock/load_static.py

```diff
@@ -0,0 +1,44 @@
+"""
+Helper class for loading generated file from webpack.
+"""
+import json
+import logging
+
+from pkg_resources import resource_string
+from django.conf import settings
+
+logger = logging.getLogger(__name__)  # pylint: disable=invalid-name
+
+
+class LoadStatic:
+    """
+    Helper class for loading generated file from webpack.
+    """
+
+    _manifest = dict()
+    _base_url = ''
+
+    @staticmethod
+    def reload_manifest():
+        """
+        Reload from manifest file
+        """
+        json_data = resource_string(__name__, 'static/dist/manifest.json').decode("utf8")
+        LoadStatic._manifest = json.loads(json_data)
+        if LoadStatic._manifest['is_dev_server']:
+            host = ''
+        elif hasattr(settings, 'LMS_ROOT_URL'):
+            host = settings.LMS_ROOT_URL
+        else:
+            host = ''
+            logger.error('LMS_ROOT_URL is undefined')
+        LoadStatic._base_url = host + LoadStatic._manifest['base_url']
+
+    @staticmethod
+    def get_url(key):
+        """
+        get url from key
+        """
+        if not LoadStatic._base_url:
+            LoadStatic.reload_manifest()
+        return LoadStatic._base_url + LoadStatic._manifest[key]
```

Comment:
Hopefully, this is okay in `stage/prod` allow CND script from ORA. If not, we might need additional configuration on the server.